client/js/components/drawer.js

```diff
@@ -0,0 +1,25 @@
+class Drawer {
+  constructor(el) {
+    this.el = el;
+    this.trigger = el.getElementsByClassName('js-trigger')[0];
+    this.trigger.addEventListener('click', this.toggleActive.bind(this));
```

Comment:
As this is a class we shouldn't have to be binding here, or if we have to, we're probably using it's scope wrong.